Jenkinsfile

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env groovy
 //Leave the above line alone.  It identifies this as a groovy script.
-@Library('vs-common-build') _
+@Library('vs-build-tools') _
 
 def lvVersions = ['2019']
 
-diffPipeline(lvVersions[0])
-ni.vsbuild.PipelineExecutor.execute(this, lvVersions)
+ni.vsbuild.PipelineExecutor.execute(this, 'veristand, 'lvVersions)
+diffPipeline(lvVersions[0])
```

Comment:
Regarding the inversion of these two lines, was this necessary? I made that change on purpose - having the diffbot run before the pipeline - so that a failed build will not block the diffbot.